src/cmd/flash.test.js

```diff
@@ -0,0 +1,36 @@
+const { expect, sinon } = require('../../test/setup');
+const FlashCommand = require('./flash');
+const usbUtil = require('./usb-util');
+const { PlatformId } = require('../lib/platform');
+
+describe('Flash Command', () => {
+	afterEach(() => {
+		sinon.restore();
+	});
+
+	describe('_getDeviceInfo', () => {
+		let device;
+		beforeEach(() => {
+			device = {
+				id: '3c0021000947343432313031',
+				platformId: PlatformId.PHOTON,
+				version: '3.3.1',
+				isInDfuMode: false
+			};
+			// sinon.stub(usbUtil, 'getOneUsbDevice').resolves(device);
+		});
+
+		it('returns information about the device', async () => {
```

Comment:
I wasn't able to stub this function by uncommenting the line 20. Every time I test it I need to connect a device and it will fail also given that the expected info doesn't match with my device

Reply:
Another thing regarding this test is even if pass the device connection still open and the test never ends.
![image](https://github.com/particle-iot/particle-cli/assets/6526979/ecb4ab77-ac13-49bd-8906-bd9cf3c13b97)

Reply:
Thanks. I rushed pushing the review button before my next meeting. I'll take another pass at these changes.